requirements-docs.txt

```diff
@@ -2,4 +2,4 @@ markdown-include~=0.6.0
 mkdocs~=1.3.0
 mkdocs-material~=8.3.8
 pygments~=2.12.0
-pymdown-extensions~=9.5
+pymdown-extensions>=10.0
```

Comment:
Does this change mean we accept any version larger or equal than `10.0.0`, including all future major versions (e.g. `12.x.x`)? If yes, I'd propose to restrict it to the current major version: `~=10.*` to prevent automatic switches to versions with potentially breaking changes.